phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/ContainerToolsTab.java

```diff
@@ -60,6 +60,13 @@ private void populate() {
             this.lockAll();
             FileChooser fileChooser = new FileChooser();
             fileChooser.setTitle(tr("Choose executable"));
+
+            // open in container directory if it exists
+            File containerDir = new File(this.container.getPath());
+            if (containerDir.canRead()) {
```

Comment:
How about we use the home directory of the user as a fallback?

Reply:
We could do that but to be honest, I don't think it makes much sense. I mean to get to this point, you must already choose an existing container so how can it be that the container directory does not exist?